WordPress/WordPressTest/CoreDataMigrationTests.m

```diff
@@ -602,4 +751,20 @@ - (NSManagedObject *)insertDummyPostInContext:(NSManagedObjectContext *)context
     return post;
 }
 
+- (void)cleanModelObjectClassnames:(NSManagedObjectModel *)model
+{
+    // NOTE: Suppose the following scenario...
+    //  -   `Model N`'s Entity X contains `Attribute A`
+    //  -   `Model N+1`'s Entity X *class* implementation differs from the one bundled in N.
+    //      Added / Removed Attributes. It could differ vastly!
+    //
+    // Problem is... whenever we test `Model N` or `Model N+1`, Core Data will always load the latest
+    // NSManagedObject subclass. This could prove troublesome in a variety of scenarios.
+    // For that reason, we're implementing this helper, which will nuke NSMO's classnames.
+    //
```

Comment:
> This could prove troublesome in a variety of scenarios.

I believe you, but after reading the comment I still don't understand what we're fixing. Perhaps add an example?